plugins/oomd.yaml

```diff
@@ -1,16 +1,19 @@
+# This is a templated file which is used by the krew-release-bot
+# from the GitHub Action to raise a pull request to the krew-index
+# (https://github.com/kubernetes-sigs/krew-index) with the correct tag
```

Comment:
I added this to[ my repository](https://github.com/jdockerty/kubectl-oomd/blob/main/.krew.yaml#L1-L3) for my own understanding, I didn't consider that the bot would also pick up the comment 🤦 

I can remove this if we want to keep the `krew-index` manifests completely clean.

Reply:
Lets do that.